source/proof_resolution/spec/proof_resolution_spec.rb

```diff
@@ -71,11 +78,19 @@
           messages: [],
           success: true,
           timed_out: false,
-          context: { stages: [
-            { resolution: LexisNexis::InstantVerify::Proofer.vendor_name },
-            { state_id: Aamva::Proofer.vendor_name },
-          ] },
-          transaction_id: nil,
+          context: {
+            stages: [
+              {
+                resolution: LexisNexis::InstantVerify::Proofer.vendor_name,
+                transaction_id: lexisnexis_transaction_id,
+              },
+              {
+                state_id: Aamva::Proofer.vendor_name,
+                transaction_id: aamva_transaction_id,
+              },
+            ],
+          },
+          transaction_id: lexisnexis_transaction_id,
```

Comment:
Is this field needed with it specified in the stages key?

Reply:
Technically no, now it's redundant.

However we are logging the existing transaction_id, so I figured it was easier to preserve that for a little more compatibility?

How strongly do you feel about removing it?

Reply:
not strongly at all, mostly curious 🙂

Reply:
Example query that can pull out transaction_id:

```
fields
  properties.event_properties.proofing_results.transaction_id,
  properties.event_properties.success,
  @timestamp, @message
| filter name = 'Doc Auth optional submitted'
| filter @message LIKE /transaction_id/
| sort @timestamp desc
| limit 20
```